src/DotNetWorker.Grpc/RpcExtensions.cs

```diff
@@ -117,9 +117,13 @@ internal static FunctionDefinition ToFunctionDefinition(this FunctionLoadRequest
             return new GrpcFunctionDefinition(loadRequest, methodInfoLocator);
         }
 
+        /// <summary>
+        /// Returns an RpcException for system exception scenarios.
+        /// </summary>
+        /// <returns>An RpcException</returns>
         internal static RpcException? ToRpcException(this Exception exception)
```

Comment:
Are we using explicit nullables? If you are interested, you can add this attribute to this function to improve compiler nullable analysis. Same with `ToUserRpcException`

``` CSharp
[return: NotNullIfNotNull("exception")]
```

Reply:
hmm seeing "Error CS0122: 'NotNullIfNotNullAttribute' is inaccessible due to its protection level" in the build when that's added - is is meant for public methods?

Reply:
We just might not be on a late enough .net or SDK version for it to be visible. That is fine - leave it out for now.